checkers.go

```diff
@@ -6,7 +6,7 @@ import (
 	"regexp"
 	"strings"
 
-	"github.com/niemeyer/pretty"
+	"github.com/kr/pretty"
```

Comment:
This is not in the list of changes. Were the fixes in the fork made upstream yet?

Reply:
Nevermind, thanks. The previous PR had the details.